pkgs/development/libraries/libgsignon-glib/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitLab, pkgconfig, automake, autoconf, libtool
+, glib, vala, gobjectIntrospection, gsignond, gettext, check
+, dbus, python3Packages, glibcLocales, docbook_xml_dtd_43
+, docbook_xml_dtd_45, docbook_xsl, gtk-doc }:
+
+stdenv.mkDerivation rec {
+  pname = "libgsignon-glib";
+  version = "2.4.1";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitLab {
+    owner = "accounts-sso";
+    repo = pname;
+    rev = version;
+    sha256 = "1h7s5ibvy0pbk9pm95a6gm293cpcqq65daszi6gl31i64mc758ap";
+  };
+
+  nativeBuildInputs = [ 
+    autoconf
+    automake
+    check
+    gettext
+    glibcLocales
+    gobjectIntrospection
+    libtool
+    pkgconfig
+    vala
+  ];
+
+  buildInputs = with python3Packages; [
+    dbus
+    docbook_xml_dtd_43
+    docbook_xml_dtd_45
+    docbook_xsl
+    glib
+    gsignond
+    gtk-doc
+    pygobject3
+    python
+  ];
+
+  preConfigure = "NOCONFIGURE=1 ./autogen.sh";
+
+  configureFlags = [
+    "--enable-introspection=yes"
+    "--disable-static"
+    "--enable-gtk-doc"
+    "--enable-dbus-type=session"
+    "--enable-python"
```

Comment:
We might want to move python to a separate output: https://github.com/NixOS/nixpkgs/commit/84fc0622cf4bfa1660b83b0b65f02cd9e7a460d4